sn_client/src/audit/dag_crawling.rs

```diff
@@ -62,26 +64,93 @@ impl Client {
         spend_processing: Option<Sender<SignedSpend>>,
         verify: bool,
     ) -> WalletResult<SpendDag> {
-        info!("Building spend DAG from {spend_addr:?}");
```

Comment:
this function shall now be feature guarded?

Reply:
Same as above, this is `sn_client`